plugin/phpactor.vim

```diff
@@ -536,7 +546,9 @@ function! phpactor#_rpc_dispatch(actionName, parameters)
     " >> collection
     if a:actionName == "collection"
         for action in a:parameters["actions"]
+            let g:_phpactorRpcActionIsCollection = v:true
             let result = phpactor#_rpc_dispatch(action["name"], action["parameters"])
```

Comment:
So you call `phpactor#_rpc_dispatch` for each action received, before the user have filter them ?

Reply:
yes -- but this is where I'm confused - it does that already (it calls the callback handler when the items are selected). But I confess to not fully understanding everything that's going on :eyes:

Reply:
For me what appen is:
* We receive the collection of action
* In the loop we dispatch each action
* The rpc handler call fzf for each of them
  * The first fzf window is open
  * Once opened the second one is open (or not because already one windows exists but it doesn't matter)
  * and again until the end of the loop

So we must not use fzf in a loop, we need to treat it like the chain I mentioned.
So that the user choose the class for the first one, and only once he made a choice we call the next action

Reply:
I got a question, let's say that we are in a file with 3 classes not yet imported: A, B and C.
The classes A and C have multiple possible FQCN and need the user to choose, B on the other hand as only one possibility.

In the collection that the new handler returns, does all three actions are the same ?
Or do we have two actions to choose the right FQCN and one to import the only possibility for B ?